tests/tests/tier0/bluechi-restart-unit/systemd/simple.service

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=Run a service for 15 seconds
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/sleep 15
```

Comment:
Is there a reason to sleep for 15s? If I understood the purpose of this unit correctly, then it should just be kept `active` forever, right? In that case I think it would be better to define the unit like this:
```
[Unit]
Description=A very simple service

[Service]
Type=simple
ExecStart=/bin/true
RemainAfterExit=yes
```

Reply:
I will try this one, but `/bin/true` exits immediately so using this with `RemainAfterExit` seems to me like quite big hack, that's why I use `/usr/bin/sleep` instead ...

Reply:
It works, but the whole events flow depends on the service you restart. For example:

* /bin/true: Active -> Inactive -> Active
* /usr/bin/sleep: Active -> Deactivating -> Inactive -> Active
* cups.service: Active -> Deactivating -> Inactive -> Activating -> Active

Luckily for the restart detection we need only the simples flow: Active -> Inactive -> Active.

Reply:
Yes, the intermediate states `Deactivating` and `Activating` don't need to be tested, I think, since it depends on the actual service and how it behaves. The important aspect about the restart is that it stops and restarts the service again - so the flow active -> inactive -> active should be checked.